cdap-ui/app/directives/group-side-panel/group-side-panel-ctrl.js

```diff
@@ -17,21 +17,30 @@
 angular.module(PKG.name + '.commons')
   .controller('MySidePanel', function ($scope) {
     this.groups = $scope.panelGroups;
-
+    this.view = $scope.view || 'icon';
+    this.add = function() {
```

Comment:
This should not be here. Will remove this function. `+` and `<<` functionalities don't really need to reside in group-side-panel. We should add them outside of it.